### ISSUE TITLE
Signal Quality over 100%?

### ISSUE THREAD

**Issue by lampmaster1** [SanderGi/WiFiStrengthTester#1]
Is the signal strength supposed to be over 100 percent?

**Comment by SanderGi** [comment]
Yes. There is no universally accepted definition for what signal quality (%) should be so I just implemented the [one proposed by Microsoft](https://docs.microsoft.com/en-us/windows/win32/api/wlanapi/ns-wlanapi-wlan_association_attributes?redirectedfrom=MSDN). Signal quality is really just a relative measure of signal strength (in the case of my program it linearly maps 0 through 100% to -100 through -50 dBm RSS). Therefore, if the RSS is greater than -50 dBm the quality is supposed to be over 100%. 

The signal quality is mostly just included for convenience (since it can help gauge how strong the network is compared to an average network). **When evaluating the signal strength of your antenna, I recommend using the signal strength (RSS) reading which is in dBm (not percent).** Since RSS is well defined and doesn't depend on implementation.

Other programs have a wide variety of implementations of signal quality (from linear to exponential scales). In the next version, I could implement the same definition of signal quality as Wifi Info View if we think that would be more useful (and if I can find documentation for it somewhere)?
